src/system/ExtensionsModule/Tests/Fixtures/maximum_composer.json

```diff
@@ -38,9 +38,12 @@
       "url": "foo",
       "oldnames": ["food"],
       "capabilities": {
-        "hook_subscriber": {"class": "Zikula\\FooModule\\Container\\HookContainer"},
-        "user": {"route": "zikulafoomodule_foo_index"},
-        "admin": {"route": "zikulafoomodule_bar_index"}
+        "admin": {
```

Comment:
`hooks_subscriber` is gone?

Reply:
yes, because we do this using service tags now

Reply:
I believe this is still used for `hasCapability` tests, etc.

Reply:
No. This is all done by collectors.

Reply:
See https://github.com/zikula/core/blob/master/src/Zikula/HookBundle/Controller/HookController.php#L68-L75

Reply:
that is for hooks but not all capabilities. see https://github.com/zikula/core/blob/master/src/system/ExtensionsModule/Api/CapabilityApi.php#L53